ansible/roles/packages/tasks/flatcar.yaml

```diff
@@ -28,3 +28,43 @@
   systemd:
     name: kubelet
     enabled: true
+
```

Comment:
Why is this stuff going in a role called `packages`?

Reply:
existing kubelet related systemd drop-ins are created in `packages` role. So I thought to keep this flex volume related systemd related plugin together in the same package. 
Do you think this can create problems for flatcar AMI built for pre-provisioned usecase?

Reply:
Nope! I think it's going to be a needed change. I didn't get why this task needed to be in `packages` but your reasoning makes sense. Maybe we could add in a comment explaining why we chose this spot